src/pages/MangaLib/main.ts

```diff
@@ -0,0 +1,300 @@
+/* eslint-disable new-cap */
+/* eslint-disable global-require */
+import { pageInterface } from '../pageInterface';
+import { SyncPage } from '../syncPage';
+import { Manga, getMangaData, getChaptersData, isPageAPI, Chapters } from '../AnimeLib/api';
```

Comment:
I merged it already. But is the bearer token really necessary? I did not see anything yet that would need it. It is a bit scary that the api is a different tld. Do you think it would be possible without it?

https://github.com/MALSync/MALSync/blob/master/src/pages/AnimeLib/api.ts#L257

Reply:
I included it just in case since I didn't know which PR you will merge first.

> But is the bearer token really necessary? I did not see anything yet that would need it

There are some manga which require you to auth to read it. There are not a lot of it but taking the token it the only thing I could think of beside implementing oauth. There's an example of said manga - https://mangalib.me/ru/manga/8359--watermelon-
Without auth you cant read it + and only API data you can get is manga info (so no chapters data). That means if user will try to read it, the whole "subchapter system" will not work, so to ensure 100% stability I did this token thing. I can remove it if you think this is too dangerous. 

> It is a bit scary that the api is a different tld

What so scary about it? You mean it could be some 3rd party api? They use this api on all they websites for manga/anime/novels. Without using API there's no way to get cover image + chapter info + chapters count on sync page